pages/impact/_year/index.vue

```diff
@@ -262,11 +271,15 @@ export default {
     }
     .credits {
         max-width: $container-l-main + px;
-        margin: var(--space-3xl) auto;
         font-style: italic;
         font-weight: normal;
-        font-size: 16px;
         line-height: 26px;
+
+        ::v-deep p,
+        ::v-deep a {
+            font-size: var(--step--1);
```

Comment:
@include step--1 
if we want to use caption styling with proxima nova

Reply:
good catch. i will update this.